tests/test_errors.py

```diff
@@ -532,30 +532,31 @@ def finalize(self, cfg: Any) -> None:
     pytest.param(
         Expected(
             create=lambda: None,
-            op=lambda cfg: OmegaConf.structured(NotOptionalInt),
+            op=lambda _: OmegaConf.structured(NotOptionalInt),
             exception_type=ValidationError,
             msg="Non optional field cannot be assigned None",
-            object_type_str=None,
-            ref_type_str=None,
+            key="foo",
+            object_type=NotOptionalInt,
+            parent_node=lambda _: {},  # dummy parent
         ),
         id="dict:create_none_optional_with_none",
     ),
     pytest.param(
         Expected(
             create=lambda: None,
-            op=lambda cfg: OmegaConf.structured(NotOptionalInt),
+            op=lambda _: OmegaConf.structured(NotOptionalInt),
             exception_type=ValidationError,
-            object_type=None,
+            object_type=NotOptionalInt,
             msg="Non optional field cannot be assigned None",
-            object_type_str="NotOptionalInt",
-            ref_type_str=None,
+            key="foo",
+            parent_node=lambda _: {},  # dummy parent
         ),
         id="dict:create:not_optional_int_field_with_none",
     ),
     pytest.param(
         Expected(
             create=lambda: None,
-            op=lambda cfg: OmegaConf.structured(NotOptionalA),
+            op=lambda _: OmegaConf.structured(NotOptionalA),
```

Comment:
In general, try to avoid no-op changes that are not directly related to code you are changing. (I know that in this case you did initially change this code so this is not a change request).

Reply:
> In general, try to avoid no-op changes that are not directly related to code you are changing.

Ok, got it.